manifests/addon.pp

```diff
@@ -34,28 +34,36 @@
 #
 
 define splunk::addon (
-  $splunk_home                   = '/opt/splunkforwarder',
+  $splunk_home                   = undef,
   $package_manage                = true,
   Optional[String] $package_name = undef,
   $inputs                        = {},
 ) {
 
+  include '::splunk::params'
```

Comment:
please remove the leading `::`. That is legacy syntax that was required for puppet 3.

Reply:
I have...a lot of these to remove lol